manifest-api-staging.yml

```diff
@@ -1,7 +1,6 @@
 ---
 
 inherit: manifest-api-base.yml
-command: scripts/run_app_paas.sh gunicorn -c /home/vcap/app/gunicorn_config.py --error-logfile /home/vcap/logs/gunicorn_error.log -w 6 -b 0.0.0.0:$PORT application
 services:
```

Comment:
Shouldn't you change the worker number rather than deleting the line here?

Reply:
This is overriding the command line in the manifest-api-base.yml file. Deleting this line means that staging will use the default.